### PR TITLE
Update error handling for insufficient funds

### LINKED ISSUE #546: Account without CRC Tokens cannot update Safe version
**Describe the bug**
In order to execute any transaction and pay for it in CRC (actually transfers, trusts ans any transaction after deployment are paid with CRC) yo need to have some funds in the account. Likewise, for updating the Safe version you need to be able to pay for the tx with CRC.

**To Reproduce**
1. Go to `devpreview`
2. Open the console tab in the browser
3. Wait for the `updateSafeVersion` function to run
4. See console error: `Uncaught (in promise) CoreError: No tokens given to pay transaction`

**Expected behavior**
I expect to see a UX message saying something like "Please send some Circles to this account if you want to perform any transaction"

### PR DESCRIPTION
**Pull request by mikozet**
Closes #546

### CODE REVIEW

File: src/utils/debug.js [hunk 1]
```diff
@@ -43,3 +44,18 @@ export default function logError(error) {
...
 }
+
+export function translateErrorForUser(error) {
+  let text;
+  if (error instanceof CoreError) {
+    if (error.code == ErrorCodes.INSUFFICIENT_FUNDS) {
+      text = translate('ErrorCodes.CoreErrorInsufficientFunds');
+    }
+  } else if (error instanceof TransferError) {
+    /*eslint-disable no-empty */
```
Comment (louilinn) [inline src/utils/debug.js]: do we want these other cases here even if we are not doing anything?

**Review by louilinn — COMMENTED** [review]
The code looks fine to me!
I take it you have tested and verified the behaviour :)

**Comment by louilinn** [comment]
I had to rebase and force push this branch

File: src/utils/debug.js (hunk 1 above)
Comment (mikozet) [reply src/utils/debug.js]: I was thinnking it can be usefull for another task with errors but I agree we can remove it

**Review by louilinn — APPROVED** [review]
I will merge for next release.
Please also confirm you tested and so on?

**Comment by mikozet** [comment]
> 

Hi yes it has been tested :)

**Pull request merged**